json-everything.net/wwwroot/md/vocabs-data-2022.md

```diff
@@ -16,66 +16,76 @@ The intent for this keyword is to cover the use cases discussed across several i
 
 The ID for this vocabulary is `https://json-everything.net/vocabs-data-2022` (the URI to this document).
 
-A draft 2020-12 meta-schema which includes this vocabulary has been defined for convenience.  The ID for the meta-schema is `https://json-everything.net/meta/data-2022`, and it can also be found at this address.
+A draft 2020-12 meta-schema which includes this vocabulary has been defined for convenience.  The ID (`$id`) for the meta-schema is `https://json-everything.net/meta/data-2022`, and it can also be found at this address.
 
-## 3. The `data` Keyword
+## 3. Definitions
 
-### 3.1 Syntax and Semantics
+### 3.1 Formed Schema
+
+The schema object created as a result of dereferencing all of the values in the `data` keyword as described in section 4.1.
+
+### 3.2 Host Schema
+
+The schema object which contains the `data` keyword.  The processing rules that govern this schema also govern the formed schema, as specified by section 4.2.
+
+## 4. The `data` Keyword
+
+### 4.1 Syntax and Semantics
 
 The value of `data` must be an object.  The keys of the object are interpreted as JSON Schema keywords, and the values MUST be one of
 
-- JSON Pointers per [RFC 6901](https://www.rfc-editor.org/rfc/rfc6901)
+- JSON Pointers per [RFC 6901](https://www.rfc-editor.org/rfc/rfc6901) 
 - [Relative JSON Pointers](https://json-schema.org/draft/2019-09/relative-json-pointer.html)
-- URI references per [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986), potentially with JSON Pointer fragments
+- URI-encoded JSON Pointer identifier per [RFC 6901, §6](https://www.rfc-editor.org/rfc/rfc6901#section-6)
```

Comment:
If instead you say "fragment-only URI using URI-encoded JSON Pointers per..." then you can get rid of the statement about parsing order further down, because fragment-only URIs, full URIs, JPs, and RJPs do not overlap at all in syntax.  Unless you're trying to support some non-fragment-only references?  But that seems confusing.

Reply:
Sure, I can make this update.  The only catch is I _am_ trying to support full URIs + JP fragments (`https://example.com/data#/foo`).